lib/rspec/rails/matchers/active_job.rb

```diff
@@ -189,6 +191,13 @@ def detect_args_signature_mismatch(jobs)
             nil
           end
 
+          def skip_signature_verification?
+            return true unless defined?(RSpec::Mocks)
```

Comment:
Trying to honour that these matchers might be used without RSpec Mocks, as indicated by this comment here:

https://github.com/rspec/rspec-rails/blob/1ba134da2c8a48c3e4a26d6bb9104ef33d91781f/lib/rspec/rails/matchers/have_enqueued_mail.rb#L1-L3

But I'm not sure if this guard is strictly necessary...

Reply:
You can remove this check, that is about avoiding conflict with other mocking frameworks, you can safely load configuration for the later check.

Reply:
Thanks for clarifying. I've removed this guard.

I tried requiring `rspec/mocks/configuration` but it doesn't seem possible to require in isolation - we then get errors regarding uninitialized constants for (Mocks) Syntax, ExampleMethods etc. Instead of requiring all of `rspec/mocks` I've left this without a require for now... happy to adjust as preferred.